dozer/cogs/levels.py

```diff
@@ -317,6 +334,82 @@ async def checkrolelevels(self, ctx):
     `{prefix}checkrolelevels`: Returns an embed of all the role levels 
     """
 
+    @group(invoke_without_command=True, aliases=["moderatelevels", "levelsmoderation"])
+    @guild_only()
+    async def adjustlevels(self, ctx):
+        """Allows for moderators to adjust a members level/xp"""
+        await ctx.send(f"Invalid subcommand\nFor help with adjustlevels use `{ctx.prefix}help adjustlevels`")
+
+    adjustlevels.example_usage = """
+    `{prefix}adjustlevels setlevel <@Snowplow or "Snowplow"> 15`:\n Sets member Snowplow's level to 15 
+    `{prefix}adjustlevels changexp <@Snowplow or "Snowplow"> -1500`:\n Changes member Snowplow's xp by -1500xp 
+    `{prefix}adjustlevels swapxp <@Snowplow or "Snowplow"> <@Dozer or "Dozer">`:\n Swaps Snowplow's xp with Dozer's xp
+    `{prefix}adjustlevels transferxp <@Snowplow or "Snowplow"> <@Dozer or "Dozer">`:\n Adds Snowplow's xp to dozer's xp
+    """
+
+    @adjustlevels.command()
+    @guild_only()
+    @has_permissions(manage_messages=True)
+    async def setlevel(self, ctx, member: discord.Member, level: int):
+        """Changes a members level to requested level"""
+        if level >= 100000:
```

Comment:
This max level which is hard coded in several places in this new code should really be a cog-wide constant.